WebViewControl/WebView.cs

```diff
@@ -138,22 +154,34 @@ public partial class WebView : IDisposable {
             chromium.JSDialogHandler = new InternalJsDialogHandler(this);
             chromium.DragHandler = new InternalDragHandler(this);
             chromium.KeyboardHandler = new InternalKeyboardHandler(this);
+        }
 
-            if (!Settings.OsrEnabled) {
-                // having the handler (by default) seems to cause some focus troubles, enable only osr disabled
-                chromium.FocusHandler = new InternalFocusHandler(this);
-            }
-
-            EditCommands = new EditCommands(chromium);
-
-            disposables = new IDisposable[] {
-                chromium,
-                AsyncCancellationTokenSource
-            };
+        private void DetachChromiumHandlers() {
```

Comment:
I dont see a reason for having this... the reports dont show problems here... you need to cut off the roots not the leafs